docs/src/tutorials/good-parts.md

```diff
@@ -0,0 +1,101 @@
+# Apalache tips: the bad and good parts
```

Comment:
Not sure if I like "the bad and good parts".
I'd say something like "Leveraging symbolic model-checking"

Reply:
yes, we should name it differently. I just want to be accurate here, because it's not always about symbolic model checking. Sometimes, it is just quirks of our encoding.

Reply:
I would call it "Apalache specification tips"

Reply:
That's better, but maybe not prominent enough to catch people's eye.
How about "Tips for specification writers: Speeding up Apalache"?

Reply:
How about "Specification golden nuggets in Apalache"? :)

Reply:
or "Tips from the Apalache Oracle"? :))